uint8 drink_id
uint8 count